src/main/java/ru/nsu/fit/schedule/port/ScheduleController.java

```diff
@@ -1,7 +1,24 @@
 package ru.nsu.fit.schedule.port;
 
-import org.springframework.web.bind.annotation.RestController;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import ru.nsu.fit.schedule.api.ScheduleService;
+
+import static ru.nsu.fit.schedule.port.ScheduleUrl.*;
 
 @RestController
+@AllArgsConstructor
+@RequestMapping(SCHEDULE_URL)
 public class ScheduleController {
+    private final ScheduleService scheduleService;
+
+    @PostMapping(RESET_URL)
+    public void resetSchedule(@RequestParam("student_id") Long studentId) {
+        scheduleService.resetSchedule(studentId);
+    }
+
+    @PostMapping(PIN_URL)
+    public void pinSchedule(@RequestParam("student_id") Long studentId, @RequestParam("group") String group) {
```

Comment:
нужно передавать айди студента и группы в теле (body) запроса. Можешь объединить их в record PinRequestDto, тогда код будет таким
```suggestion
    public void pinSchedule(@RequestBody PinRequestDto pinRequestDto) {
```